src/main/resources/application.yaml

```diff
@@ -216,3 +216,13 @@ task:
   check-jurisdiction-log-in:
     check-validity-duration: PT5M # only ensure this often that no log-in attempt is rejected by IDAM
 
+logging:
+  level:
+    uk:
+      gov:
+        hmcts:
+          reform:
+            bulkscan:
+              orchestrator:
+                services:
+                  ccd: ${CCD_API_LOG_LEVEL:INFO}
```

Comment:
we may want to log all debug levels in app ?

Reply:
I think 
`logging:
  level:
    uk:
      gov:
        hmcts:
          reform:`